src/libraries/Common/src/Interop/Windows/Kernel32/Interop.ReadDirectoryChangesW.cs

```diff
@@ -28,13 +28,8 @@ internal struct FILE_NOTIFY_INFORMATION
             internal FileAction Action;
 
             // Note that the file name is not null terminated
-            private uint FileNameLength;
-            private char _FileName;
-
-            internal unsafe ReadOnlySpan<char> FileName
-            {
-                get { fixed (char* c = &_FileName) { return new ReadOnlySpan<char>(c, (int)FileNameLength / sizeof(char)); } }
-            }
+            internal readonly uint FileNameLength;
+            // internal readonly char FileName[1];
```

Comment:
Nit: Remove comment?

Reply:
This field is part of the struct definition in Windows SDK. I have added comment to make it clear.